backend/src/apiserver/client_manager.go

```diff
@@ -127,6 +134,31 @@ func (c *ClientManager) init() {
 
 	c.swfClient = client.CreateScheduledWorkflowClientOrFatal(
 		getStringConfig(podNamespace), getDurationConfig(initConnectionTimeout))
+
+	port, err := strconv.Atoi(getStringConfig(mysqlServicePort))
+	if err != nil {
+		glog.Fatalf("Failed to parse valid MySQL service port from %q: %v", getStringConfig(mysqlServicePort), err)
+	}
+
+	cfg := &mlpb.ConnectionConfig{
+		Config: &mlpb.ConnectionConfig_Mysql{
+			&mlpb.MySQLDatabaseConfig{
+				Host:     proto.String(getStringConfig(mysqlServiceHost)),
+				Port:     proto.Uint32(uint32(port)),
+				Database: proto.String("mlmetadata"),
+				User:     proto.String("root"),
+			},
+		},
+	}
+
+	mlmdStore, err := mlmetadata.NewStore(cfg)
```

Comment:
Is there support for a "fake" implementation using SQLLite so that we can run a lot of backend tests as go tests?

Reply:
Is it possible to add support for SQLLite right away so that we can run tests quickly locally?

Reply:
Nope, sorry, I can't do it right now. 
If it's any consolation, I plan to move all of this out to its own service soon, so API server can stay the same as before.